HW4/Zoriana007/Homework 17.03.22 Task 2 (4 lecture).py

```diff
@@ -0,0 +1,9 @@
+n = int(input("Please input number: "))
```

Comment:
list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)